### PR TITLE
Chore/details

### PR DESCRIPTION
**Pull request by Cristianjs93**
# Chore details
Details in error handling and component rendering were corrected.

### CODE REVIEW

File: src/components/StickyNavigation/index.jsx [hunk 1]
```diff
@@ -14,32 +14,33 @@ const StickyNavigation = ({ menu }) => {
...
+                </h5>
               </li>
-            ))}
-          </div>
-        ))}
+              {item.products.map((product, subIndex) => (
+                <li key={product.id} className='sticky-nav__inner-list__item'>
+                  <h6
+                    className={
+                      index == 0 && subIndex == 0
```
Comment (ht1204) [inline src/components/StickyNavigation/index.jsx]: `index === 0 && subIndex === 0`

File: src/components/StickyNavigation/index.jsx [hunk 2]
```diff
@@ -14,32 +14,33 @@ const StickyNavigation = ({ menu }) => {
...
+        {menu &&
+          menu.map((item, index) => (
+            <div key={index} className='sticky-nav__box'>
+              <li className='sticky-nav__list__item'>
+                <h5
                   className={
-                    index == 0 && subIndex == 0
-                      ? 'sticky-nav__inner-list__item__product--selected'
-                      : 'sticky-nav__inner-list__item__product'
+                    index == 0
```
Comment (ht1204) [inline src/components/StickyNavigation/index.jsx]: `index === 0`

File: src/components/StickyNavigation/index.jsx (hunk 1 above)
Comment (Cristianjs93) [reply src/components/StickyNavigation/index.jsx]: resolved

File: src/components/StickyNavigation/index.jsx (hunk 2 above)
Comment (Cristianjs93) [reply src/components/StickyNavigation/index.jsx]: resolved

**Review by ht1204 — APPROVED** [review]

**Pull request merged**